catalog/core/catalog-core-downloadaction/src/main/java/org/codice/ddf/catalog/download/action/ResourceDownloadActionException.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.download.action;
+
+public class ResourceDownloadActionException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    public ResourceDownloadActionException(String message) {
+        super(message);
+    }
+
+    public ResourceDownloadActionException(String message, Throwable throwable) {
+        super(message, throwable);
+    }
+
+    public ResourceDownloadActionException(Throwable throwable) {
```

Comment:
Do we need this constructor? Not sure we should allow someone to create one of those without a message.